<launch>

  <node name="generate_command_autodetect_joy"   pkg="kondo_b3mservo_rosdriver" type="generate_command_autodetect_joy.py" output="screen"/>
  <node name="multi_servo_control" pkg="kondo_b3mservo_rosdriver" type="multi_servo_control.py" output="screen">
    <rosparam param="servo_id">
      <!-- input the pid gain parameters to the list below -->
      [3,4]
    </rosparam>
    <rosparam param="control_mode">
      <!-- input the pid gain parameters to the list below -->
      [0,16]
    </rosparam>
  </node>

  <node name="joy_node"   pkg="joy" type="joy_node">
    <param name="autorepeat_rate" value="100"/>
  </node>

</launch>
